test/fuzz/fuzz-unit-file/directives.scope

```diff
@@ -1,2 +1,60 @@
-scope
```

Comment:
The fuzzer uses the first line to figure out what it should load
```
        if (read_line(f, LINE_MAX, &p) < 0)
                return 0;

        t = unit_type_from_string(p);
        if (t < 0)
                return 0;

        if (!unit_vtable[t]->load)
                return 0;
```
I'm not sure `[Scope]` can be used here instead of `scope`.

Reply:
Hah, thank you, completely missed this. In that case I tweaked the generation script a bit and merged `Service`, `Unit`, and `Install` sections into a single `.service` file to to match the `UnitType` mapping.

Reply:
I think it should cover all the keys. Thanks!

It would be great to help the fuzzer to discover possible values with dictionaries: https://github.com/mirrorer/afl/blob/2fb5a3482ec27b593c57258baae7089ebdc89043/docs/README#L266 but unfortunately I'm not sure it can be done automatically.